src/EnergyPlus/OutputProcessor.cc

```diff
@@ -3929,43 +2989,43 @@ namespace OutputProcessor {
 
         // Facility indices are in the 100s
         if (has(meterName, "Electricity:Facility")) {
-            state.dataOutputProcessor->indexGroupKey = 100;
+            return 100;
```

Comment:
What are these things used for ?

Reply:
Based on the values, I assume they are HTTP return codes.  :laughing: